src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -100,7 +100,7 @@ class GenericClient[F[_]: Monad, _](applicationName: String)(
     new ThreeDAssetMappings(requestSession, modelId, revisionId)
 }
 
-final case class Client(applicationName: String)(
+final case class Client(applicationName: String, baseUri: String)(
```

Comment:
This should have `"api.cognitedata.com"` as a default value, so the user doesn't have to type it out for the normal use case.

Reply:
However, see comments above about "Auth" and doing something similar.